### PR TITLE
Feature/add co xlate

### PR DESCRIPTION
**Pull request by cosentino-smeup**
## Description
In this branch has been implemented the operation code XLATE. 

Related to:
- T10_A90_01
- T10_A90_02
- T10_A90_03

## Checklist:
- [x] There are tests regarding this feature
- [x] The code follows the Kotlin conventions (run `./gradlew ktlintCheck`)
- [x] The code passes all tests (run `./gradlew check`)
- [ ] There is a specific documentation in the `docs` directory

### CODE REVIEW

File: rpgJavaInterpreter-core/src/test/resources/smeup/T10_A90.rpgle [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+     D A90_A5          S              8
+     D £DBG_Pas        S              3
+     D £DBG_Str        S            100
+     D NNN             S              6  0 INZ(100000)
+     D A90_UP          C                   'ABCDEFGHIJKLMNOPQRS-
+     D                                     TUVWXYZ'
+     D A90_LO          C                   'abcdefghijklmnopqrs-
+     D                                     tuvwxyz'
+      *
+     C*                   EVAL      £DBG_Pas='P01'
```
Comment (lanarimarco) [inline rpgJavaInterpreter-core/src/test/resources/smeup/T10_A90.rpgle:13]: Why is this step commented?

**Review by lanarimarco — COMMENTED** [review]
I have an answer?

File: rpgJavaInterpreter-core/src/test/resources/smeup/T10_A90.rpgle (hunk 1 above)
Comment (cosentino-smeup) [reply rpgJavaInterpreter-core/src/test/resources/smeup/T10_A90.rpgle]: I created from mulangt90 but I noticed afterward that there was a case not managed. So I commented the code that didn't work. It should be fixed in the next commit.

File: rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/parsing/parsetreetoast/misc.kt [hunk 2]
```diff
@@ -1282,6 +1285,25 @@ internal fun CsCHECKContext.toAst(conf: ToAstConfiguration): Statement {
     )
 }
 
+private fun FactorContext.toDoubleExpression(conf: ToAstConfiguration, index: Int): Pair<Expression, Int?> =
```
Comment (lanarimarco) [inline rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/parsing/parsetreetoast/misc.kt:1288]: I don't understand why this function returns a pair, is it really necessary?

File: rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/parsing/parsetreetoast/misc.kt [hunk 3]
```diff
@@ -1282,6 +1285,25 @@ internal fun CsCHECKContext.toAst(conf: ToAstConfiguration): Statement {
...
+    if (this.text.contains(":")) this.text.toDoubleExpression(toPosition(conf.considerPosition), index, conf) else this.content.toAst(conf) to null
+
+private fun String.toDoubleExpression(position: Position?, index: Int, conf: ToAstConfiguration): Pair<Expression, Int?> {
```
Comment (lanarimarco) [inline rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/parsing/parsetreetoast/misc.kt:1291]: See above

**Review by lanarimarco — COMMENTED** [review]
Another question

File: rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/parsing/parsetreetoast/misc.kt (hunk 2 above)
Comment (cosentino-smeup) [reply rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/parsing/parsetreetoast/misc.kt:1288]: I'll remove it in the next commit

File: rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/parsing/parsetreetoast/misc.kt (hunk 3 above)
Comment (cosentino-smeup) [reply rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/parsing/parsetreetoast/misc.kt:1291]: I'll remove it in the next commit

**Review by lanarimarco — COMMENTED** [review]
You need to resolve conflict

**Review by lanarimarco — APPROVED** [review]
Ok, next time remember the format of PR comment.

**Pull request merged**